admin/controllers/pupil-pin.js

```diff
@@ -95,7 +95,9 @@ const postGeneratePins = async (req, res, next) => {
   }
   let school
   try {
+    // OLD code - writes to check table
     await checkStartService.prepareCheck(pupilsList, req.user.School, req.user.schoolId, pinEnv)
```

Comment:
The comments suggest that the old code is still running? Is this correct?

Reply:
yes